m365/powerbi/README.md

```diff
@@ -0,0 +1,116 @@
+## **SCuBA PowerBI Integration Steps**
+
+### **Supported Scenarios**
+
+This guide covers **two primary scenarios**:
+
+1. **PowerBI with Virtual Network Gateway:**
+   - Configure PowerBI to use a Virtual Network Gateway to access data from the ScubaConnect storage account.
+
+2. **PowerBI Desktop Direct Access:**
+   - Use PowerBI Desktop to retrieve and publish data directly to a PowerBI workspace. If the report needs to be viewed by others this is the ideal option. If others don't need to view the report, see below.
+   - You could also keep the data local and not publish to a PowerBI workspace.
+
+> [!NOTE]
+> **Virtual Network Gateway (Scenario 1) is not available in GCC environments.**<br>
+> It is supported only in Commercial and GCCHigh.
+> [Limitations](https://learn.microsoft.com/en-us/data-integration/vnet/overview#limitations)
+
+### **Pre-Requisites**
+Ensure the following requirements are met before proceeding to avoid setup issues.
+
+1. **Virtual Network Gateway Requirements**
+   > **Note:** The Virtual Network Gateway option is only available in Commercial and GCCHigh environments. It is not supported in GCC.
+   - [Register resource provider](https://learn.microsoft.com/en-us/data-integration/vnet/create-data-gateways#step-1-register-microsoftpowerplatform-as-a-resource-provider)
+     - `Microsoft.PowerPlatform`
+   - [Subnet delegation](https://learn.microsoft.com/en-us/data-integration/vnet/create-data-gateways#step-2-associate-the-subnet-to-microsoft-power-platform)
+     - `Microsoft.PowerPlatform/vnetaccesslinks`
+   - [License Requirements](https://learn.microsoft.com/en-us/data-integration/vnet/overview#limitations)
+   - [PowerBI Desktop application](https://www.microsoft.com/en-us/download/details.aspx?id=58494&msockid=101dc2aa8ee969a80209d6378f076840)
+
+2. **Requirements for GCC Environments**
+   - [PowerBI Desktop application](https://www.microsoft.com/en-us/download/details.aspx?id=58494&msockid=101dc2aa8ee969a80209d6378f076840)
+
+---
+
+### **General Setup Steps**
+
+1. **Download and Import the PowerBI Report**
+   - Download the PowerBI report from the repository and import it into PowerBI Desktop.
+     - [Path to Download](https://github.com/cisagov/ScubaConnect/blob/main/m365/powerbi/SCuBA%20M365%20Report%20(Azure%20Blob%20Storage).pbix)
+
+2. **Modify the Data Sources**
+   - From the **Home** tab, select **Transform data** then **Data source settings**.
+     - Highlight the Azure Blob Storage path and select **Change Source**.
+       - Enter your blob storage account name and select **OK**.
+     - Highlight the Azure Blob Storage path and select **Edit Permissions**.
```

Comment:
Seems like this was missing the step to set the `OutputDirectory`. I'm not too sure how PowerBI works, but I found that drilling down.

Might help to give example values as a reference for each item here